region/lowernorfair/east/Red Kihunter Shaft.json

```diff
@@ -813,7 +813,7 @@
       "requires": [
         {"notable": "KiHunter Dodge"},
         "h_canNavigateHeatRooms",
-        "canTrickyJump",
+        "canTrickyDodgeEnemies",
```

Comment:
This would push this up to Expert (the `Kihunter Manipulation from Middle Door` is also in expert)
looking at the video, it looked like expert